src/api/app/controllers/webui/request_controller.rb

```diff
@@ -181,9 +181,8 @@ def diff
   end
 
   def list_small
-    required_parameters :project # the minimum
     redirect_to(user_show_path(User.current)) && return unless request.xhr? # non ajax request
-    requests = BsRequest.list(params)
+    requests = params[:project] ? BsRequest.list(params) : []
```

Comment:
Why isn't this in the routes?

Reply:
Because it's coming from an AJAX call where the project is added by an autocomplete.

Reply:
And how do you produce an ajax call without this parameter?

Reply:
We can get rid of this check. In case an ajax call is received without this `project` parameter we can rely on the exception thrown by `BsRequest.list`. Removing the check.